http/src/main/resources/reference.conf

```diff
@@ -414,7 +412,7 @@ helmConfig {
         name = "ca-secret"
         secretFiles = [
           {
-            name = "ca-crt"
+            name = "ca.crt"
```

Comment:
For whatever reason, the nginx ingress controller wants dots in these names, and I can't figure out how to change it.

https://kubernetes.github.io/ingress-nginx/examples/auth/client-certs/